<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>950</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777000</width>
    <height>16777000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 224, 215);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>901</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="MapOptionsScreen" name="mapOptionsScreen" native="true"/>
     </item>
     <item>
      <widget class="MapScreen" name="mapScreen" native="true"/>
     </item>
     <item>
      <widget class="SettingsScreen" name="settingsScreen" native="true"/>
     </item>
     <item>
      <widget class="ParseScreen" name="parseScreen" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #CCBCAD;</string>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actiondynamic_map"/>
    <addaction name="actionvector_map"/>
    <addaction name="actionsettings"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actiondownload_osm">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>download osm</string>
   </property>
  </action>
  <action name="actionparse_osm">
   <property name="text">
    <string>parse osm</string>
   </property>
  </action>
  <action name="actionset_map_options">
   <property name="text">
    <string>set map options</string>
   </property>
  </action>
  <action name="actionsettings">
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="actiondynamic_map">
   <property name="text">
    <string>dynamic map</string>
   </property>
  </action>
  <action name="actionvector_map">
   <property name="text">
    <string>vector map</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParseScreen</class>
   <extends>QWidget</extends>
   <header>parsescreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapOptionsScreen</class>
   <extends>QWidget</extends>
   <header>mapoptionsscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapScreen</class>
   <extends>QWidget</extends>
   <header>mapscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsScreen</class>
   <extends>QWidget</extends>
   <header>settingsscreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
